### ISSUE TITLE
The instrument URL doesn't have enough contrast on isis and isis study card dataview

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/datagateway#898]
**Description**:
## Dark mode 

The instrument URL doesn't have enough  contrast to satisfy WCAG 2 AA. Suggested foreground colour change to #86b4ff
<br><br>

![url_cardview](https://user-images.githubusercontent.com/83226114/139669561-53958dcf-ec49-4db3-83c8-a201761bcee1.png)

**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done
